src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.includes(':'))
```

Comment:
Filtering styles that include a colon is a good approach to ensure only valid declarations are processed. However, consider handling cases where the colon might be misplaced or there are multiple colons, even though it's not required by the task.